ee/tabby-webserver/src/handler.rs

```diff
@@ -30,37 +30,55 @@ use crate::{
     path::db_file,
     repositories,
     schema::{
-        auth::AuthenticationService, create_schema, repository::RepositoryService, Schema,
-        ServiceLocator,
+        auth::AuthenticationService, create_schema,
+        github_repository_provider::GithubRepositoryProviderService, repository::RepositoryService,
+        Schema, ServiceLocator,
+    },
+    service::{
+        create_service_locator, event_logger::create_event_logger,
+        new_github_repository_provider_service,
     },
-    service::{create_service_locator, event_logger::create_event_logger},
     ui,
 };
 
 struct RepositoryAccessImpl {
-    service: Arc<dyn RepositoryService>,
+    repository_service: Arc<dyn RepositoryService>,
+    github_service: Arc<dyn GithubRepositoryProviderService>,
```

Comment:
```suggestion
    git_repository_service: Arc<dyn RepositoryService>,
    github_repository_service: Arc<dyn GithubRepositoryProviderService>,
```

Also update other namings correspondingly